lms/templates/instructor/instructor_dashboard_2/cohort-form.underscore

```diff
@@ -95,7 +95,7 @@
                                             <%=
                                             interpolate(
                                                 // Translators: Any text between %(screen_reader_start)s and %(screen_reader_end)s is only read by screen readers and never shown in the browser.
-                                                '%(screen_reader_start)sWarning:%(screen_reader_end)s The previously selected content group was deleted. Select another content group.',
+                                                gettext('%(screen_reader_start)sWarning:%(screen_reader_end)s The previously selected content group was deleted. Select another content group.'),
```

Comment:
@talbs @andy-armstrong the `gettext` was not included in these lines. Please be sure that you're properly testing your features for i18n.